app/web/features/communities/events/EventForm.tsx

```diff
@@ -118,18 +120,24 @@ export default function EventForm({
   const { t } = useTranslation([GLOBAL, COMMUNITIES]);
   const classes = useEventFormStyles();
 
-  const {
-    control,
-    errors,
-    handleSubmit,
-    getValues,
-    register,
-    setValue,
-    watch,
-    formState: { dirtyFields },
-  } = useForm<CreateEventData>();
+  const now = dayjs();
+  const defaultDate = now.get("hour") === 23 ? now.add(1, "day") : now;
+
+  const { date: eventStartDate, time: eventStartTime } =
+    splitTimestampToDateAndTime(event?.startTime);
+  const { date: eventEndDate, time: eventEndTime } =
+    splitTimestampToDateAndTime(event?.endTime);
+
+  const timeDelta = useRef(60);
+  const defaultEndTime = useMemo(
+    () =>
+      dayjs()
+        .add(1, "hour")
+        .add(timeDelta.current, "minutes")
+        .format("HH:[00]"),
+    []
+  );
```

Comment:
I see this is used a couple of times, maybe worth moving to a utility function?

Also I'm not entirely following the default date/time logic here. For `defaultDate`, why does it not suffice to add 1 hour, won't that roll over to the next day?

Or maybe it's just simpler to make the default start date/time 24 hours ahead from now?

Reply:
I didn't change the logic, just moved it, so I rather leave that for now and make a separate ticket.

What I did do was set defaultValues in the useForm and move the defaultValues up to the parent component. It was throwing an error about the `.startOf` dates being undefined in the child component. I'm guessing the order of stuff rendering caused this with the new automatic batching for React v18. At least that's my best guess.